ee/clickhouse/queries/funnels/funnel_time_to_convert.py

```diff
@@ -0,0 +1,56 @@
+from ee.clickhouse.queries.funnels.funnel import ClickhouseFunnelNew
+
+
+class ClickhouseFunnelTimeToConvert(ClickhouseFunnelNew):
+    def run(self, *args, **kwargs) -> list:
+        if len(self._filter.entities) == 0:
+            return []
+
+        return self.perform_query()
+
+    def perform_query(self):
```

Comment:
True for here, as well as trends: I don't see a good reason to have the `perform_query` function. It creates an extra "weird" codepath when we don't need it to?

For example, the `Base`'s `run()` method is sufficient everywhere? It calls `_exec_query()`, then `_format_results()`, both of which we can override as need be, but we don't need to override `run()` - unless it's doing something totally different. (which so far, I don't think it is, so fitting into the same existing execution model is very clean)

Reply:
Agreed. The additional overrides were a holdover from `ClickhouseFunnelTrends`, whose formatting works a bit differently (I inherited that part from the previous query – probably ought to be refactored some time), but in this case overriding only `_format_results` is sufficient and better.